src/llamacrawl/cli/commands/query.py

```
@@ -376,6 +376,4 @@ def _parse_date(date_str: str) -> datetime:
     except ValueError:
         pass
 
-    raise ValueError(
-        f"Could not parse date '{date_str}'. Use YYYY-MM-DD or ISO 8601 format."
-    )
+    raise ValueError(f"Could not parse date '{date_str}'. Use YYYY-MM-DD or ISO 8601 format.")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)